project/AppDependencies.scala

```diff
@@ -8,7 +8,7 @@ object AppDependencies {
     "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "4.3.0",
     "uk.gov.hmrc" %% "play-frontend-govuk" % "0.71.0-play-27",
     "uk.gov.hmrc" %% "simple-reactivemongo" % "8.0.0-play-27",
-    "uk.gov.hmrc" %% "auth-client" % "5.4.0-play-27",
+    "uk.gov.hmrc" %% "auth-client" % "5.7.0-play-27",
```

Comment:
auth-client is included as part of bootstrap so shouldn't need to be added here. The latest version of bootstrap has auth-client 2.6.0 though.